.github/main.workflow

```diff
@@ -1,41 +0,0 @@
-workflow "Python 3.7 - tox" {
```

Comment:
Please revert all changes to this file.

Reply:
That file also appears to have been deleted from the dev branch here: https://github.com/home-assistant/home-assistant/tree/dev/.github
Do I still need to do anything with this?

Reply:
If you can do a rebase so that it doesn't show any changes to that file (I know it's deleted, but we're trying to keep a clean git history), that would be ideal.